common/src/chain/config/mod.rs

```diff
@@ -245,6 +245,16 @@ impl ChainConfig {
         }
     }
 
+    #[must_use]
+    pub fn vrf_public_key_address_prefix(&self) -> &'static str {
+        match self.chain_type {
+            ChainType::Mainnet => "mvrfpk",
+            ChainType::Testnet => "tvrfpk",
+            ChainType::Regtest => "rvrfpk",
+            ChainType::Signet => "svrfpk",
+        }
```

Comment:
Should the wallet use these now for displaying the VRF public key? I don't know if we do display it for any reason.

Reply:
changed the VRF public key printing in the wallet commands